ethcore/src/spec/spec.rs

```diff
@@ -98,6 +98,8 @@ pub struct CommonParams {
 	pub block_reward: U256,
 	/// Registrar contract address.
 	pub registrar: Address,
+	/// Apply reward
+	pub apply_reward: bool,
```

Comment:
Also an assumption. I agree, that not zero bock_reward without its application doesn't make much sense. But is it safe to rely on such things?